packages/database/data-migrations/20240611153820_pricing_v2/data-migration.ts

```diff
@@ -0,0 +1,172 @@
+import { Prisma, PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+type TSubscriptionStatusLegacy = "active" | "cancelled" | "inactive";
+
+interface TSubscriptionLegacy {
+  status: TSubscriptionStatusLegacy;
+  unlimited: boolean;
+}
+
+interface TFeatures {
+  inAppSurvey: TSubscriptionLegacy;
+  linkSurvey: TSubscriptionLegacy;
+  userTargeting: TSubscriptionLegacy;
+  multiLanguage: TSubscriptionLegacy;
+}
+
+interface TOrganizationBillingLegacy {
+  stripeCustomerId: string | null;
+  features: TFeatures;
+}
+
+async function main() {
+  await prisma.$transaction(
+    async (tx) => {
+      //   const startTime = Date.now();
+      console.log("Starting data migration for pricing v2...");
+
+      // Free tier
+      const orgsWithoutBilling = await tx.organization.findMany({
+        where: {
+          billing: {
+            path: ["stripeCustomerId"],
+            equals: Prisma.AnyNull,
+          },
+        },
+      });
+
+      console.log(
+        `Found ${orgsWithoutBilling.length} organizations without billing information. Moving them to free plan...`
+      );
+
+      for (const organization of orgsWithoutBilling) {
+        await tx.organization.update({
```

Comment:
Can we add all of them to a Promise and await Promise.all() to be able to update a lot of orgs in the limited time